packages/terra-badge/tests/jest/__snapshots__/Badge.test.jsx.snap

```diff
@@ -1,20 +1,20 @@
 exports[`test should render a badge component in the order, icon and text with medium size 1`] = `
 <span
-  className="terra-Badge terra-Badge--medium terra-Badge--default">
+  className="badge medium default">
   <img
     alt="Test icon" />
   <span
-    className="terra-Badge-text">
+    className="text">
```

Comment:
You can view source of the badge here to see what I'm referring to https://terra-core-deployed-pr-519.herokuapp.com/static/#/site/badge